### PR TITLE
Add to list from list - core journey

### PR DESCRIPTION
**Pull request by laurieboyes**
Clicking this button takes you to the same form you see when you save an article in the first place, so you can add it to another list.
Appears on article cards within lists and within saved articles.
Behind myftLists flag.

screen shot 2015-11-19 at 12 24 58

### CODE REVIEW

File: views/partials/list-card/partials/actions.html [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+				data-trackable="remove-from-list"
+				type="submit"
+				aria-label="Remove"
+				title="Remove"
+				>Remove
+		</button>
+	</form>
+{{/if}}
+<form class="n-myft-ui"
+	  method="GET"
```
Comment (phamann) [inline views/partials/list-card/partials/actions.html]: What is this weird hack :stuck_out_tongue_winking_eye:

**Comment by phamann** [comment]
:+1:

**Comment by keirog** [comment]
:+1:

File: views/partials/list-card/partials/actions.html (hunk 1 above)
Comment (laurieboyes) [inline views/partials/list-card/partials/actions.html]: :shame:

**Pull request merged**